static/templates/actions_popover_content.handlebars

```diff
@@ -1,7 +1,7 @@
 {{! Contents of the "message actions" popup }}
 <ul class="nav nav-list actions_popover pull-right">
     <li>
-        <a href="#" class="popover_edit_message" data-message-id="{{message.id}}">
+        <a href="#" class="popover_edit_message" data-message-id="{{message.id}}" title="{{t 'Edit' }} (i then Enter)">
```

Comment:
This one doesn't quite work to document this way.  We probably just want a first-class hotkey for it.